### ISSUE TITLE
elm-test init creates old folder structure

### ISSUE THREAD

**Issue by dschinkel** [elm-explorations/test#29]
Just trying to get this working over v19.

I noticed that even though:

- I have the v19 Elm **binary** installed for OS X
- I'm running Elm 19
- I've got an `elm.json` at the root of my project

<img width="346" alt="screen shot 2018-09-18 at 6 26 50 pm" src="https://user-images.githubusercontent.com/338798/45722094-70eb4c00-bb70-11e8-84e4-d937c6ce8a42.png">

**Problem**: When I run `elm-test init` it creates a `test` directory but in that directory I only see:

`elm-package.json` (why isn't it **elm.json** ??????)
`Example.elm`

For troubleshooting, ran `npm install -g elm-test` again and it appears to have installed the new elm-test for v19 as you can see at the bottom...although I don't know if the prior lines in the install results there are "normal":

<img width="893" alt="screen shot 2018-09-18 at 6 22 30 pm" src="https://user-images.githubusercontent.com/338798/45722016-194ce080-bb70-11e8-967d-8dc2fabe0c6b.png">

However after running `elm-test init`, something isn't right here.

```
Created /Users/daveschinkel/Code/katas/minMax/elm-minmax-kata/tests
Created /Users/daveschinkel/Code/katas/minMax/elm-minmax-kata/tests/elm-package.json
Created /Users/daveschinkel/Code/katas/minMax/elm-minmax-kata/tests/Example.elm
/Users/daveschinkel/Code/katas/minMax/elm-minmax-kata/.gitignore already exists
/bin/sh: elm-package: command not found
Unhandled exception while running the tests: { Error: Command failed: elm-package install --yes
    at checkExecSyncError (child_process.js:603:11)
    at Object.execSync (child_process.js:640:13)
    at runElmTest (/usr/local/lib/node_modules/elm-test/lib/elm-test.js:291:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/elm-test/lib/elm-test.js:742:1)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
  status: 127,
  signal: null,
  output: [ null, null, null ],
  pid: 14555,
  stdout: null,
  stderr: null }
```

**That creates a test folder but with an elm-package.json in it, not elm.json.**

**Comment by dschinkel** [comment]
didn't see this, I'll try it https://github.com/elm-explorations/test/issues/28

**Comment by dschinkel** [comment]
resolved using #28
